backend/pyproject.toml

```diff
@@ -31,9 +31,11 @@ trio = "^0.18.0"
 uvicorn = "^0.14.0"
 uvloop = "^0.15.2"
 vyper-config = "1.0.0"
+opensearch-py = "^2.8.0"
```

Comment:
I've considered this before, since we could use a much more recent Opensearch; but the backend uses `Elasticsearch` and I don't think it's a great idea to use both...

Reply:
removed `opensearch-py`

Reply:
That makes sense. I also removed any references to `opensearchpy`.